index.d.ts

```diff
@@ -464,8 +464,2262 @@ interface Typography {
 
 export interface Theme {}
 
-export const defaultTheme: Theme
-export const classicTheme: Theme
+export interface DefaultTheme extends Theme {
+  tokens: {
+    colors: {
+      gray900: string
+      gray800: string
+      gray700: string
+      gray600: string
+      gray500: string
+      gray400: string
+      gray300: string
+      gray200: string
+      gray100: string
+      gray90: string
+      gray75: string
+      gray50: string
+      blue900: string
+      blue800: string
+      blue700: string
+      blue600: string
+      blue500: string
+      blue400: string
+      blue300: string
+      blue200: string
+      blue100: string
+      blue50: string
+      blue25: string
+      red700: string
+      red600: string
+      red500: string
+      red100: string
+      red25: string
+      green900: string
+      green800: string
+      green700: string
+      green600: string
+      green500: string
+      green400: string
+      green300: string
+      green200: string
+      green100: string
+      green25: string
+      orange700: string
+      orange500: string
+      orange100: string
+      orange25: string
+      purple600: string
+      purple100: string
+      teal800: string
+      teal100: string
+      yellow800: string
+      yellow100: string
+      muted: string
+      default: string
+    }
+    fontFamilies: {
+      display: string
+      ui: string
+      mono: string
+    }
+    text: {
+      300: {
+        fontSize: string
+        fontWeight: number
+        lineHeight: string
+        letterSpacing: string
```

Comment:
I noticed there are some existing types in a `Typography` interface that have some of these values - I didn't do any interface reduction/abstraction, this was a pretty raw conversion from the current object shape. I can go back and try to consolidate some of these common properties into other interfaces if we think it's worth it. What do ya'll think?